api/src/test/java/org/openmrs/validator/AllergyValidatorTest.java

```diff
@@ -178,4 +179,18 @@ public void validate_shouldPassForAValidAllergy() {
 		
 		assertFalse(errors.hasErrors());
 	}
+
+	@Test
+	public void valid_shouldRejectNumericReactionValue() throws Exception {
```

Comment:
Remove the unwanted spaces in this method!

Reply:
@HerbertYiga ......indeed I also wanted to remove them but......according to the styling of the other Tests........... I just wanted to follow the Order............. Is it worth it.......to remove them or......just leave them as the other tests